install.sh

```
@@ -0,0 +1,526 @@
+    if [[ -n "$CHECKSUM_URL" ]]; then
+      print_info "3. Verify checksum from: ${CHECKSUM_URL}"
+    fi
+    print_info "4. Extract and install binary to: ${INSTALL_DIR}/quetty"
+    print_info "5. Set executable permissions"
+    check_path
+    exit 0
+  fi
+
+  # Create temporary directory
+  TEMP_DIR=$(mktemp -d)
+  trap "rm -rf '$TEMP_DIR'" EXIT
```

Shellcheck (0.10.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)